### ISSUE TITLE
Tabs lacks WCAG support for reading back tab status

### ISSUE THREAD

**Issue by dpouliot** [filamentgroup/collapsible#69]
it should alert the user if the current tab is active. via `role=tab` and `aria-selected`

**Comment by zachleat** [comment]
Merged! Thank you!
